### DISCUSSION TITLE
Comparative neuroimaging

### DISCUSSION THREAD

**Discussion by pcklink** [PRIME-RE/prime-re.github.io#68] [Collaborations]
This may need to be split up in multiple sub-projects later, but it's a start.

**Comment by beruss** [comment]
Does anyone know of a digital Cebus atlas?

**Reply by afni-dglen** [reply]
Late posting, so you may have already found it. I ran across a couple capuchin datasets available here, but you need to sign a simple agreement with them. I don't see any reference to an atlas parcellation though.

[https://www.j-monkey.jp/BIR/
](https://www.j-monkey.jp/BIR/
)
See this article about this repository:
[https://www.sciencedirect.com/science/article/pii/S1053811923002422
](https://www.sciencedirect.com/science/article/pii/S1053811923002422
)
